package.json

```diff
@@ -81,35 +85,35 @@
     "yargs": "^15.0.1"
   },
   "scripts": {
-    "convertToModules": "gulp convertToModules",
+    "convertToModules": "gulp -f gulpfile.cjs convertToModules",
     "start": "node server.cjs",
     "startPublic": "node server.cjs --public",
-    "build": "gulp build",
-    "build-watch": "gulp build-watch",
-    "buildApps": "gulp buildApps",
-    "clean": "gulp clean",
-    "cloc": "gulp cloc",
-    "combine": "gulp combine",
-    "combineRelease": "gulp combineRelease",
-    "coverage": "gulp coverage",
-    "generateDocumentation": "gulp generateDocumentation",
-    "generateDocumentation-watch": "gulp generateDocumentation-watch",
+    "build": "gulp -f gulpfile.cjs build",
```

Comment:
We can revert all of the `-f gulpfile.cjs` and just tell gulp to use it by default, see https://github.com/CesiumGS/cesium/pull/8541/files#diff-a663dcab4db6e595455e0c837b86ee03

Reply:
Nice, done.